base_images/Dockerfile.bamboo

```diff
@@ -5,6 +5,7 @@ LABEL maintainer="Rizbi Hassan" \
 
 ENV NODE_VERSION="16.x"
 ENV TERRAFORM_VERSION "0.13.6"
+ENV PYTHON_VERSION "3.9.10"
```

Comment:
I would pull Python version 3.9.16 as it is the lateset 3.9 series with bug fixes.

Reply:
updated